plugins/mobile-plugin/pom.xml

```diff
@@ -26,23 +30,39 @@
     <dependency>
       <groupId>io.appium</groupId>
       <artifactId>java-client</artifactId>
-      <version>6.0.0-BETA5</version>
+      <version>6.0.0</version>
```

Comment:
Fill free to update to 7.0 after merging https://github.com/sbtqa/page-factory-2/pull/166